README.md

```diff
@@ -9,7 +9,7 @@ forever to find the sources, so all sources are added post my realization.
 
 # Cites
```

Comment:
is 'Cites' a typo that also needs to be fixed?

eg 'Sites' instead of 'Cites' or is this intentional? If not, I'd also include this as this commit addresses typos.